src/MainPage/ChannelsList.module.scss

```diff
@@ -1,111 +1,103 @@
-.channelsListContainer  {
```

Comment:
Zadanie domowe, w ChannelsList masz trzy powtarzające się requesty, poczytaj o Axios i pomyśl jak możnaby było zoptymalizować te requesty, żeby zabierały mniej miejsca. Zrobimy to na następnych zajęciach.